plugins/flytekit-ray/flytekitplugins/ray/task.py

```diff
@@ -76,10 +75,6 @@ def pre_execute(self, user_params: ExecutionParameters) -> ExecutionParameters:
                 "excludes": ["script_mode.tar.gz", "fast*.tar.gz"],
             }
 
-            cfg = self._task_config
```

Comment:
cc @fiedlerNr9 mind taking a look at this?

Reply:
LGTM